include/odp/api/spec/ipsec.h

```diff
@@ -71,6 +71,48 @@ typedef enum odp_ipsec_op_mode_t {
 
 } odp_ipsec_op_mode_t;
 
+/**
+ * IPSEC TEST SA parameter type
+ */
+typedef enum odp_ipsec_test_sa_param_type_t {
+	/** Next sequence number value */
+	ODP_IPSEC_TEST_SA_SEQ_NUM = 0,
+
+	/** Highest authenticated sequence number */
+	ODP_IPSEC_TEST_SA_ANTIREPLAY_WINDOW_TOP
+
+} odp_ipsec_test_sa_param_type_t;
+
+/**
+ * IPSEC TEST SA parameter value
+ */
+typedef union odp_ipsec_test_sa_param_val_t {
+	/** Next sequence number value */
+	uint64_t seq_num;
+
+	/** Highest authenticated sequence number */
+	uint64_t antireplay_window_top;
+
+} odp_ipsec_test_sa_param_val_t;
+
+/**
+ * IPSEC TEST SA parameters to allow applications to override unexposed IPSEC
```

Comment:
The term "override" sounds to me that the value given either permanently overrides the normal state parameter (in which case e.g. sequence number would stay constant) or once (after which the normal state parameter is used again). I think simple "set" would be better here and in other parts of the API description.

Reply:
Agreed. Updated v5 to use the term "set" instead of "override".